### PR TITLE
T/gitupdatesource

### PR DESCRIPTION
**Pull request by gilleyj**
RE: https://github.com/easybib/issues/issues/4112

### CODE REVIEW

**Comment by gilleyj** [comment]
the chef recpies seem to work and the script seem to work but needs testing in real environment it's too late for me right now

**Comment by gilleyj** [comment]
Till not sure if you want to take this or give it to @fh or @baccenfutter but i think you owned the original ticket

File: stack-qa/recipes/deploy-vagrant-ci.rb [hunk 1]
```diff
@@ -55,6 +55,26 @@
...
+cookbook_file '/opt/vagrant/bin/update_github_sources.sh' do
+  source 'update_github_sources.sh'
+  owner deploy_user_name
+  group deploy_user_name
+  mode '0755'
+  action :create
+end
+
+# setup git_update_script cron to run as root, script will pick up user with git creds
+cron 'cookbooks_report' do
```
Comment (till) [inline stack-qa/recipes/deploy-vagrant-ci.rb]: @gilleyj Can you use the `cron_d` provider thingy? It'll create this in `/etc/cron.d/` as a job vs. editing the crontab directly.

https://github.com/till/easybib-cookbooks/tree/master/cron#resources-and-providers

**Comment by till** [comment]
@gilleyj Thanks! I :eyes: and had one minor thing. Otherwise great!

**Comment by gilleyj** [comment]
the cron was changed to cron_d according to doc's everything else is same-same for settings/params.  Should be good then?

**Pull request merged**